automat/_introspection.py

```diff
@@ -6,22 +6,24 @@
 
 
 def copycode(template, changes):
-    names = [
-        "argcount", "nlocals", "stacksize", "flags", "code", "consts",
-        "names", "varnames", "filename", "name", "firstlineno", "lnotab",
-        "freevars", "cellvars"
-    ]
-    if hasattr(code, "co_kwonlyargcount"):
-        names.insert(1, "kwonlyargcount")
-    if hasattr(code, "co_posonlyargcount"):
-        # PEP 570 added "positional only arguments"
-        names.insert(1, "posonlyargcount")
-    values = [
-        changes.get(name, getattr(template, "co_" + name))
-        for name in names
-    ]
-    return code(*values)
-
+    if hasattr(code, "replace"):
+        return template.replace(**{"co_" + k : v for k, v in changes.items()})
+    else:
```

Comment:
Since the `if` statement returns, the `else` and the `indent` is redunant. Avoiding the extra indent makes it easier to verify that the diff does not change the old path. Would you mind doing that?